dev/scripts/sortPackageJson.js

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env node
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * This script sorts:
+ * 
+ * package.json
+ *  - activationEvents
+ *  - contributes.commands
+ *  - contributes.menus
+ * 
+ * package.nls.json
+ */
+
+const path = require('path');
+const existsSync = require('fs').existsSync;
```

Comment:
Should be able to replace these with:

```
import * as path from 'path';
import * as fs from 'fs/promises';
```

In general, we usually use imports the main reason being
> 3. Import statements are asynchronous
Being synchronous or asynchronous might not play a major role in small-scale applications. But, if we think of large-scale applications, there are hundreds of modules are used. So,
if you use require, modules will be loaded and processed one by one.
On the other hand, import statements solve this issue by working asynchronously, and they are known to perform well compared to require functions in large-scale applications.

Also just stylistically, I prefer it.